modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -1,6 +1,7 @@
 // Module included in the following assemblies:
 //
 // * nodes/nodes-scheduler-taints-tolerations.adoc
+// * post_installation_configuration/node-tasks.adoc
 
 [id="nodes-scheduler-taints-tolerations-about_{context}"]
 = Understanding taints and tolerations
```

Comment:
line 82， I think the statement is not correct:
However, if a pod that to be evicted has the `tolerationSeconds` parameter, the pod is not evicted until that time period expires.
It should be: 
However, if a pod that do tolerate the taint has the `tolerationSeconds` parameter, the pod is not evicted until that time period expires.

Reply:
line 95, for "Here, if this pod is running but does not have a matching taint", I think it means the node has a taint which the pod doesn't tolerate, if so , the pod will be evicted immediately.